src/main/java/uk/gov/hmcts/reform/hmc/helper/GetHearingActualsResponseMapper.java

```diff
@@ -37,7 +37,7 @@ public class GetHearingActualsResponseMapper extends GetHearingResponseCommonCod
 
     public HearingActualResponse toHearingActualResponse(HearingEntity hearingEntity) {
         val response = new HearingActualResponse();
-        response.setHmcStatus(getHearingStatus(hearingEntity));
```

Comment:
We should be able to delete the private `getHearingStatus()` from within this class now too!